README.md

```diff
@@ -1,2 +1,101 @@
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do list app [React version]
+
+Remember the to do list app? It is the app that helps someone to plan for the day activities so that he/she may track the progress which increases productivity. This is the React version of it
+
+## Built With
+
+- HTML and CSS
+- React.js
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+
+
+## Project Status
+
+This project is finished and deplyed with Github pages.
+
+## Screenshoots
+![Screenshot (28)](https://user-images.githubusercontent.com/90222110/155112486-9ef8f556-e195-4f94-a9a9-189161fdfe30.png)
+
+
+
+## Live Demo link
+
+
+
+## Issues
```

Comment:
Well done👏👏, You've  done a great job 👍👍 on the Readme.file, its very descriptive but there is one more thing you might want to add that you left out. You didn't add the live demo link after deploying the page, this is so important as it helps other's to see what you've done without cloning the repository on the local machine.

Reply:
Yeah, I understand it. I am going to deploy it. Thank you.